src/d3a/setup/jira/custom_storage_strategy.py

```diff
@@ -56,8 +56,8 @@ def decrease_energy_price_over_ticks(self, market):
         # example for determining the current tick number:
         current_tick_number = self.area.current_tick % self.area.config.ticks_per_slot
         if current_tick_number >= 0:
-            if reduced_rate < self.final_selling_rate:
```

Comment:
Why did you rename only this file (there is also test_strategy_custom_p.py and *_load.py).
IMO this does not have to be renamed.